docs-v2/components/TopNav.tsx

```diff
@@ -30,13 +30,13 @@ const TopNav: React.FC<Props> = () => {
       paddingX={majorScale(5)}
     >
       <Pane display="flex" alignItems="center" width={236}>
-        <Pane
+        <a href="/"><Pane
```

Comment:
Use a `Link` here from `next/Link`. It'll get transformed the same way at build time, but good to have the routing abstraction in case we want to static deploy vs. SPA deploy